### ISSUE TITLE
Some UnitTesting tests are failing with ST builds >= 4000

### ISSUE THREAD

**Issue by rwols** [sublimelsp/LSP#797]
The failing tests are

```
test_window_without_configs (test_configs.WindowConfigTests) ... FAIL
test_additional_edits (test_completion.QueryCompletionsTests) ... FAIL
test_dash_missing_from_label (test_completion.QueryCompletionsTests) ... FAIL
test_edit_after_nonword (test_completion.QueryCompletionsTests) ... FAIL
test_edit_before_cursor (test_completion.QueryCompletionsTests) ... FAIL
test_implement_all_members_quirk (test_completion.QueryCompletionsTests) ... FAIL
test_resolve_for_additional_edits (test_completion.QueryCompletionsTests) ... FAIL
test_simple_inserttext (test_completion.QueryCompletionsTests) ... FAIL
test_simple_label (test_completion.QueryCompletionsTests) ... FAIL
test_space_added_in_label (test_completion.QueryCompletionsTests) ... FAIL
test_var_prefix_added_in_insertText (test_completion.QueryCompletionsTests) ... FAIL
test_var_prefix_added_in_label (test_completion.QueryCompletionsTests) ... FAIL
test_var_prefix_using_label (test_completion.QueryCompletionsTests) ... FAIL
```

My installed packages:

```
		"AutomaticPackageReloader",
		"AvrAssembler",
		"ChaiScript",
		"CMake",
		"Dockerfile Syntax Highlighting",
		"FileIcons",
		"fish",
		"GitSavvy",
		"HexViewer",
		"OpenGL Shading Language (GLSL)",
		"Package Control",
		"PackageDev",
		"Protobuf Syntax Hightlighting",
		"SideBarTools",
		"Terminus",
		"TypeScript Syntax",
		"UnitTesting"
```

**Comment by Narretz** [comment]
Where did you get Sublime Text v >4000 from?

**Comment by rwols** [comment]
@Narretz see this post on the forums https://forum.sublimetext.com/t/emmet-2-0-alpha/47933/10

**Comment by Narretz** [comment]
Thankls!

**Comment by randy3k** [comment]
Ha, I am involved in 4/17 of your packages. Maybe I spent too much time in Sublime Text. 🤓

**Comment by rwols** [comment]
fixed by #866
